common/lib/protocol-definitions/api-report/protocol-definitions.api.md

```diff
@@ -282,20 +282,20 @@ export interface ISequencedDocumentAugmentedMessage extends ISequencedDocumentMe
 
 // @public
 export interface ISequencedDocumentMessage {
-    clientId: string;
+    clientId: string | null;
```

Comment:
we've been putting this off for a long while, but it shouldn't be that bad to consume, as the code needs to handle it already.

Reply:
Is `null` correct? Just double-checking.

Reply:
yes. it is. its what the server currently sends over the wire, the current typing is wrong to not include null.